src/styles/reset.scss

```diff
@@ -0,0 +1,10 @@
+body,
+h3,
+p {
+  margin: 0;
+}
+
+ul {
```

Comment:
Replace this tag up to body, h3, and p. Delete rule with padding, you don't need to reset padding.

Reply:
ul has default padding